src/generic-container/started-generic-container.ts

```diff
@@ -100,8 +100,8 @@ export class StartedGenericContainer implements StartedTestContainer {
     return this.inspectResult.networkSettings[networkName].ipAddress;
   }
 
-  public exec(command: Command[]): Promise<ExecResult> {
-    return execContainer(this.container, command);
+  public exec(command: Command[], options?: Dockerode.ExecStartOptions): Promise<ExecResult> {
```

Comment:
We don't expose dockerode options directly, instead we create/export our own so that people don't have to import transitive dependencies